installers/metrics/ansible/README.md

```diff
@@ -0,0 +1,15 @@
+# Crunchy Data PostgreSQL Operator Metrics Playbook
+
+<p align="center">
+  <img width="150" src="../../../crunchy_logo.png" alt="Crunchy Data"/>
+</p>
+
+Latest Release: 4.4.0
+
+## General
+
+This repository contains Ansible Roles for deploying the metrics stack for the
+Crunchy PostgreSQL Operator.
+
+See the [official Crunchy PostgreSQL Operator documentation](https://crunchydata.github.io/postgres-operator/stable/)
```

Comment:
Would this need to be the `access.crunchydata.com` link?

Reply:
Switched to the `access.crunchydata.com` link.